x-pack/plugins/security_solution/public/flyout/left/header.tsx

```diff
@@ -44,14 +46,15 @@ export const PanelHeader: VFC<PanelHeaderProps> = memo(({ selectedTabId, setSele
     ));
 
   return (
-    <EuiFlyoutHeader hasBorder>
-      <EuiTabs
-        size="l"
-        expand
-        css={css`
-          margin-bottom: -25px;
-        `}
-      >
+    <EuiFlyoutHeader
+      hasBorder
+      css={css`
+        background-color: ${euiTheme.colors.lightestShade};
+        padding-bottom: 0 !important;
+        border-block-end: none !important;
```

Comment:
this removes the bottom, or we'd have a dual line like in this screenshot
![Screenshot 2023-09-22 at 5 59 35 PM](https://github.com/elastic/kibana/assets/17276605/303af824-25bb-4a6d-b50a-2cdfa25714a2)